### PR TITLE
purescript-web-mercator

### PR DESCRIPTION
**Pull request by safareli**

### CODE REVIEW

File: example/map-halogen/src/Main.purs [hunk 1]
```diff
@@ -52,7 +54,7 @@ ui =
       [ HH.slot MapSlot mapComponent unit $ Just <<< H.action <<< HandleMapUpdate
       , HH.button
           [ HP.class_ (HH.ClassName "goto")
-          , HE.onClick (HE.input_ $ GoTo $ makeLngLat 44.81647122397245 41.661632116606455) 
+          , HE.onClick (HE.input_ $ GoTo $ LngLat.make { lng: 44.81647122397245, lat: 41.661632116606455 })
           ]
           [ HH.text "GoTo Tbilisi" ]
```
Comment (sectore) [inline example/map-halogen/src/Main.purs]: It's funny to read this ^ in every place :)

File: example/map-halogen/src/Main.purs [hunk 2]
```diff
@@ -18,7 +18,9 @@ import Halogen.HTML.Events as HE
 import Halogen.HTML.Properties as HP
```
Comment (sectore) [inline example/map-halogen/src/Main.purs]: Trying to run `bower i` in `example/map-halogen/` throws following thing:
```
bower i
...
bower purescript-react-map-gl#*          ENOTFOUND Package purescript-react-map-gl not found
```

File: example/map/src/Main.purs [hunk 3]
```diff
@@ -1,22 +1,25 @@
 module Main where
```
Comment (sectore) [inline example/map/src/Main.purs]: Similar issue while trying to install deps in `example/map` as described in https://github.com/f-o-a-m/purescript-react-map-gl/pull/5/files#r188905079

File: bower.json [hunk 4]
```diff
@@ -12,7 +12,8 @@
     "purescript-console": "^3.0.0",
     "purescript-react": "^5.1.0",
     "purescript-generics-rep": "^5.4.0",
-    "purescript-record": "^0.2.6"
+    "purescript-record": "^0.2.6",
+    "purescript-web-mercator": "f-o-a-m/purescript-web-mercator#master"
```
Comment (sectore) [inline bower.json]: Let's tag ps-web-mercator w/ a version to avoid to use every future changes of `master`.

**Comment by safareli** [comment]
added the tag and removed alternative version of building examples, (now they can only be built from root)

File: example/map-halogen/src/Main.purs (hunk 1 above)
Comment (safareli) [reply example/map-halogen/src/Main.purs]: 🕺

**Pull request merged**